karapace/kafka_rest_apis/consumer_manager.py

```diff
@@ -34,7 +34,11 @@ def new_name() -> str:
 class ConsumerManager:
     def __init__(self, config: dict) -> None:
         self.config = config
-        self.hostname = f"http://{self.config['advertised_hostname']}:{self.config['port']}"
+        self.protocol = self.config["advertised_protocol"]
```

Comment:
hey, do we need this new attribute? could it be just a standalone variable, or using `self.config["advertised_protocol"]` directly in the f-string?

If we don't need the attribute anywhere I would remove it, rit makes refactoring a bit easier.

Reply:
Yeah, moving it to the f-string is better, I have changed that, thanks for pointing out this.